src/Annotation/ApiResource.php

```diff
@@ -274,128 +312,127 @@ final class ApiResource
      *
      * @var array
      */
+    private $openapiContext;
+
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     * @see https://api-platform.com/docs/core/default-order/#overriding-default-order
+     *
+     * @var array
+     */
     private $order;
 
     /**
      * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
      *
+     * @var string|false
+     */
+    private $output;
```

Comment:
This one is https://api-platform.com/docs/core/dto/#specifying-an-input-or-an-output-data-representation